phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/engines/EnginePanel.java

```diff
@@ -104,7 +101,8 @@ private void populateContent() {
         progressSpacer.getStyleClass().add("engineSpacer");
         VBox.setVgrow(progressSpacer, Priority.NEVER);
 
-        this.setCenter(new VBox(informationContentPane, spacer, buttonBox, progressSpacer));
+        final VBox content = new VBox(informationContentPane, spacer, buttonBox, progressSpacer);
+        this.setCenter(content);
```

Comment:
I believe it would be ok to initialize the VBox on the fly in `this.setCenter()`.